src/api/registry.js

```diff
@@ -160,16 +160,23 @@ export async function setAddress(name, address) {
   return () => Resolver.setAddr(namehash, address).send({ from: account })
 }
 
-export async function setContent(name, content) {
+export async function setOnion(name, content) {
   let account = await getAccount()
   const namehash = getNamehash(name)
   const resolverAddr = await getResolver(name)
   const { Resolver } = await getResolverContract(resolverAddr)
   return await estimateAndSend(
-    Resolver.setContent(namehash, content),
+    Resolver.setText(namehash, 'onion', content),
```

Comment:
@Arachnid for hackathon purpose, I suggested the team to put protocol name into key, but this could collide with people using the keyword `onion` for something else. The team could not use contenthash despite onion does exist on [multicodec lookup table](https://github.com/multiformats/multicodec/blob/master/table.csv#L87) because the value does not fit into 32bit. Should we have some sort of standard on text field or should we extend contenthash to have bigger length?

Reply:
Not sure I understand; contenthash is variable length. What wouldn't fit into what?